app/database/schema.py

```diff
@@ -0,0 +1,102 @@
+from sqlalchemy import Column, Integer, String, Float, Boolean
+from sqlalchemy.ext.declarative import declarative_base
+
+
+Base = declarative_base()
+
+
+class BuildStateSchema(Base):
+    """Describes the state of a Build"""
+    __tablename__ = 'builds'
+
+    build_id = Column(Integer, primary_key=True, autoincrement=True)
+    completed = Column(Boolean)
+
+
+class BuildMetaSchema(Base):
```

Comment:
Why are BuildStateSchema/BuildMetaSchema/BuildArtifactSchema/etc. separate? There are several tables whose rows have a with a one-to-one correspondence with a build -- I'm interested in why you chose to split them apart.

I'm still working through reviewing this and adding comments, but here's some interesting reading in the meantime 😉:
https://martinfowler.com/bliki/AnemicDomainModel.html